compiler/luci/export/src/CircleTensorExporter.cpp

```diff
@@ -352,6 +354,50 @@ flatbuffers::Offset<circle::Buffer> encodeOpBufferByDType(FlatBufferBuilder &bui
   return CreateBuffer(builder, array_offset);
 }
 
+template <>
```

Comment:
I think memory structure comment would be helpful. Otherwise, below `count + 2` things would look like magic number.